dartagnan/src/main/java/com/dat3m/dartagnan/program/event/core/Event.java

```diff
@@ -74,7 +74,7 @@ public String getSourceCodeFile() {
 
 	public Event setCFileInformation(int line, String file) {
 		this.cLine = line;
-		this.sourceCodeFile = file;
+		this.sourceCodeFile = file.contains("/") ? file.substring(file.lastIndexOf("/") + 1) : file;
 		return this;
 	}
```

Comment:
I don't think this is a good idea. You are just dropping information here just because you do not need/want it at some other component. We should never drop free information :).

We should rename this to `sourceCodeFilePath` (it's a whole path after all).
And then we either provide an additional `getSourceCodeFileName` which returns just the file name rather than the whole path, or you add this feature directly into the ` CallStackComputation` class, i.e., it retrieves the file path, extracts the file name and then adds it to the call stack string.